TP/steps/19-form-validation-solution/src/app/shared/badge/sfeir-badge.directive.ts

```diff
@@ -19,7 +25,7 @@ export class SfeirBadgeDirective implements OnInit {
       this.renderer.setProperty(
         this.elementRef.nativeElement,
         'innerHTML',
-        '<i class="material-icons">supervisor_account</i>'
+        this.sanitizer.sanitize(SecurityContext.HTML, '<i class="material-icons">supervisor_account</i>')
```

Comment:
question: Sanitizer is used by the view to sanitize dangerous values. Why this value is dangerous ?

Reply:
On this particular example, the HTML is not dangerous, but the question of XSS issue has been asked during the formation. So we thought that it is a good  way to bring the subject on the table with an example.